pkg/utils/yaml_utils.go

```
@@ -38,65 +43,124 @@
+	y, err := ConvertToYAML(data, YAMLOptions{Indent: indent})
 	if err != nil {
 		return err
 	}
 
-	atmosConfig := ExtractAtmosConfig(data)
-	highlighted, err := HighlightCodeWithConfig(y, atmosConfig)
+	highlighted, err := HighlightCodeWithConfig(y, *atmosConfig, "yaml")
 	if err != nil {
-		// Fallback to plain text if highlighting fails
 		PrintMessage(y)
 		return nil
```

golangci-lint (1.64.8)

[error] error is not nil (line 75) but it returns nil

(nilerr)